docs/guides/composition-revisions-example.md

```diff
@@ -0,0 +1,302 @@
+  name: vpc-staging
+spec:
+  id: vpc-staging
+  compositionRevisionSelector:
+    matchLabels:
+      channel: staging
+```
+
+Expected Output:
+```shell
+myvpc.aws.example.upbound.io/vpc-staging created
+``` 
+
+## Verify the Composite Resources
+Verify the Composite Resource with the label `channel: staging` doesn't have a `REVISION`.  
+All other XRs have a `REVISION` matching the created Composition Revision.
+```shell
+kubectl get composite -o=custom-columns=NAME:.metadata.name,SYNCED:.status.conditions[0].status,REVISION:.spec.compositionRevisionRef.name,POLICY:.spec.compositionUpdatePolicy,MATCHLABEL:.spec.compositionRevisionSelector.matchLabels
+```
+Expected Output:
+```shell
+NAME          SYNCED   REVISION                                POLICY      MATCHLABEL
+vpc-auto      True     myvpcs.aws.example.upbound.io-ad265bc   Automatic   <none>
+vpc-dev       True     myvpcs.aws.example.upbound.io-ad265bc   Automatic   map[channel:dev]
+vpc-man       True     myvpcs.aws.example.upbound.io-ad265bc   Manual      <none>
+vpc-staging   False    <none>                                  Automatic   map[channel:staging]
+``` 
+
+{{< hint "note" >}}
+The `vpc-staging` XR label doesn't match any existing Composition Revisions.
+{{< /hint >}}
+
+## Create new Composition revisions
+
+### Update the Composition label
+Update the `Composition` label to `channel: staging`:
+```shell
+kubectl label composition myvpcs.aws.example.upbound.io channel=staging --overwrite
+```
+Expected Output:
+```shell
+composition.apiextensions.crossplane.io/myvpcs.aws.example.upbound.io labeled
+``` 
+
+#### Verify the updated Composition revisions
+Verify that Crossplane creates a new Composition revision:
+```shell
+kubectl get compositionrevisions -o=custom-columns=NAME:.metadata.name,REVISION:.spec.revision,LABEL:.metadata.labels.channel
+```
+Expected Output:
+```shell
+NAME                                    REVISION   LABEL
+myvpcs.aws.example.upbound.io-727b3c8   2          staging
+myvpcs.aws.example.upbound.io-ad265bc   1          dev
+``` 
+
+Crossplane assigns the Composite Resources `vpc-auto` and `vpc-staging` to Composite revision:2.  
+XRs `vpc-man` and `vpc-dev` are still assigned to the original revision:1:
+
+```shell
+kubectl get composite -o=custom-columns=NAME:.metadata.name,SYNCED:.status.conditions[0].status,REVISION:.spec.compositionRevisionRef.name,POLICY:.spec.compositionUpdatePolicy,MATCHLABEL:.spec.compositionRevisionSelector.matchLabels
+```
+Expected Output:
+```shell
+NAME          SYNCED   REVISION                                POLICY      MATCHLABEL
+vpc-auto      True     myvpcs.aws.example.upbound.io-727b3c8   Automatic   <none>
+vpc-dev       True     myvpcs.aws.example.upbound.io-ad265bc   Automatic   map[channel:dev]
+vpc-man       True     myvpcs.aws.example.upbound.io-ad265bc   Manual      <none>
+vpc-staging   True     myvpcs.aws.example.upbound.io-727b3c8   Automatic   map[channel:staging]
+``` 
+
+{{< hint "note" >}}
```

Comment:
Same issue with rendering here.

Reply:
this will be fixed on the docs site, please see the `Hints and alert boxes` chapter [here](https://docs.crossplane.io/master/contributing/docs/#code-of-conduct).

Reply:
You should be able to run `make docs.run` to render the page locally. Because of how versions work against a branch name, the page will be at http://localhost:1313/comp-rev-tutorial/guides/composition-revisions-example/